src/Raven.Server/Documents/Handlers/Processors/Documents/AbstractDocumentHandlerProcessorForGet.cs

```diff
@@ -201,15 +202,16 @@ string CreatePerformanceHintDetails()
 
         HttpContext.Response.Headers[Constants.Headers.Etag] = "\"" + result.Etag + "\"";
 
-        return await WriteDocumentsByIdResultAsync(context, metadataOnly, clusterWideTx, result);
+        return await WriteDocumentsByIdResultAsync(context, metadataOnly, clusterWideTx, result)
+                        .ConfigureAwait(false);
     }
 
     private async ValueTask<(long NumberOfResults, long TotalDocumentsSizeInBytes)> WriteDocumentsByIdResultAsync(
         TOperationContext context, bool metadataOnly, bool clusterWideTx, DocumentsByIdResult<TDocumentType> result)
     {
         long numberOfResults;
         long totalDocumentsSizeInBytes;
-        await using (var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream()))
+        var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream());
```

Comment:
In places where you have `IAsyncDisposable`, I think this would be more readable 
```
var writer = new AsyncBlittableJsonTextWriter(context, RequestHandler.ResponseBodyStream());
await using (writer.ConfigureAwait(false))
{
...
}
```

Reply:
We would avoid tremendous amount of line changes because of that

Reply:
> That would be best:
> 
> ```cs
>         await using (AsyncBlittableJsonTextWriter.Create(context, RequestHandler.ResponseBodyStream(), out var writer).ConfigureAwait(false))
>         {
>         }
> ```

Does that work?

Reply:
It does, and it is even better, we would not even need the `.ConfigureAwait(false)`

Reply:
why we do not want to add `.ConfigureAwait(false)`? Am I missing sth? It's for DisposeAsync?